bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/RegExpCharSet.java

```diff
@@ -71,7 +71,7 @@ public String stringValue(BLink parent) {
         StringJoiner atoms = new StringJoiner("");
         for (Object atom : this.charSetAtoms) {
             if (atom instanceof RegExpCharSetRange || atom instanceof RegExpCharSet
-                    || atom instanceof RegExpLiteralCharOrEscape) {
+                    || atom instanceof RegExpLiteralCharOrEscape || atom instanceof RegExpCharacterClass) {
```

Comment:
```suggestion
                    || atom instanceof RegExpLiteralCharOrEscape) {
```
We don't need this right?

Reply:
This is needed because special meta characters will be rewritten to character class causing a nested character class`[\s]`